ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/search/search.html

```diff
@@ -1,8 +1,19 @@
 <sly data-sly-use.model="com.valtech.aem.saas.api.fulltextsearch.SearchModel"
-     data-sly-test.isEdit="${wcmmode.edit}"></sly>
+     data-sly-test.isEdit="${wcmmode.edit}"
+     data-sly-test.connectionFailedAlert="${model.connectionFailedAlert}"></sly>
+<coral-alert data-sly-test="${isEdit && connectionFailedAlert}"
+             size="L"
+             variant="${connectionFailedAlert.variant.value}">
+    <coral-alert-header>${'com.valtech.aem.saas.core.search.apiConnectionFail.text' @ i18n}</coral-alert-header>
+    <coral-alert-content data-sly-test.messages="${connectionFailedAlert.messages}">
+        <ul data-sly-list.message="${messages}">
+            <li>${message}</li>
+        </ul>
+    </coral-alert-content>
+</coral-alert>
 <sly data-sly-use.template="/apps/saas-aem-module/shared/htl/templates/placeholder.html"
      data-sly-call="${template.placeholder @ isEmpty=isEdit}"></sly>
 <div data-sly-test="${isEdit}"
      data-sly-resource="${'search-tabs' @ resourceType='wcm/foundation/components/responsivegrid'}"></div>
 <search data-sly-test="${!isEdit}"
-        data-search-config="${model.configJson}"></search>
+        data-search-config="${model.configJson}"></search>
```

Comment:
new line, please adjust ide settings

Reply:
this one is intentional, for improved readability

Reply:
new line at end of file I mean